h_docs/polybft_setup.md

```diff
@@ -30,7 +30,7 @@ I am describing our custom process, because it is different.
 We need to set native token to be mintable, so we can premine balances to different addresses
 
 ```
-./hydra genesis --block-gas-limit 10000000 --epoch-size 10 --validators-path ./ --validators-prefix test-chain- --consensus polybft --native-token-config Hydra:HDR:18:true:0x211881Bb4893dd733825A2D97e48bFc38cc70a0c --reward-wallet 0x61324166B0202DB1E7502924326262274Fa4358F:1000000 --premine 0x211881Bb4893dd733825A2D97e48bFc38cc70a0c:70000000000000000000000 --premine 0xdC3312E368A178e24850C6dAC169646c5fD14b93:700000000000000000000 --proxy-contracts-admin 0x211881Bb4893dd733825A2D97e48bFc38cc70a0c --chain-id 8844
+./hydra genesis --block-gas-limit 10000000 --epoch-size 10 \ --validators-path ./ --validators-prefix test-chain- --consensus polybft --native-token-config Hydra:HDR:18:true:0x211881Bb4893dd733825A2D97e48bFc38cc70a0c   --reward-wallet 0x61324166B0202DB1E7502924326262274Fa4358F:1000000 --premine 0x211881Bb4893dd733825A2D97e48bFc38cc70a0c:70000000000000000000000 --premine 0xdC3312E368A178e24850C6dAC169646c5fD14b93:30000000000000000000000 --proxy-contracts-admin 0x211881Bb4893dd733825A2D97e48bFc38cc70a0c --chain-id 8844
```

Comment:
Why are `\` and an empty space on the next line added?

Reply:
Well, it is for sure by mistake. Will delete it